nixos/doc/manual/release-notes/rl-2111.section.md

```diff
@@ -320,3 +320,5 @@ To be able to access the web UI this port needs to be opened in the firewall.
 - GNOME desktop environment now enables `QGnomePlatform` as the Qt platform theme, which should avoid crashes when opening file chooser dialogs in Qt apps by using XDG desktop portal. Additionally, it will make the apps fit better visually.
 
 - `rofi` has been updated from '1.6.1' to '1.7.0', one important thing is the removal of the old xresources based configuration setup. Read more [in rofi's changelog](https://github.com/davatorium/rofi/blob/cb12e6fc058f4a0f4f/Changelog#L1).
+
+- Bare minimum module for `kratos` have been defined
```

Comment:
Nit:

```suggestion
- Bare minimum module for `kratos` have been defined.
```

Reply:
Unable to commit in GithubUI -> Commited in [`6b52f53` (#137164)](https://github.com/NixOS/nixpkgs/pull/137164/commits/6b52f5333a2ffe5aff32b90e05e1d3d3c5688f90)